frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -1001,6 +1002,26 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
             <dd>
               <ResourceLink kind={referenceForModel(ClusterVersionModel)} name={cv.metadata.name} />
             </dd>
+            <dt>{t('public~Upstream configuration')}</dt>
```

Comment:
"Upstream configuration" sounds odd to me. Maybe "Update server" or "Upstream update server"? @wking @megan-hall opinion?

I would use `DetailsItem` so that you get the API doc when you click on the label. Here's an example:

https://github.com/jamestalton/ocp-console/blob/e12a4adb5c4bde317d510a0590c88f6b99c2f6d6/frontend/public/components/pod.tsx#L677-L679

Reply:
I have switched to using a shared component that utilizes `DetailsItem`

Reply:
@florkbr Shall we update `Upstream configuration` to something else or keep as it is?

Reply:
Sorry @spadgett just seeing your comment now - I honestly will default to you or Trevor here in terms of wording.

Reply:
@yapei We can leave it as "Upstream configuration" for now and always revisit later. Thanks!